HW_1/task_2.py

```diff
@@ -0,0 +1,19 @@
+
+# Петя, Катя и Сережа делают из бумаги журавликов.
+# Вместе они сделали S журавликов. Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа
+# сделали одинаковое количество журавликов, а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+# *Пример:*
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+# 60 -> 10  40  10
+
+sum = int(input('Сколько всего журавликов было сделано: '))
+
+if sum % 6 == 0:
+    p = int(sum / 6)
```

Comment:
Тут проще использовать оператор целочисленного отделения.
```suggestion
    p = sum // 6
```